lambdas/sqs-message-remover/tests/test-index.js

```diff
@@ -179,17 +185,35 @@ test('sqsMessageRemover lambda does nothing for a workflow message when eventSou
   assertInvalidSqsQueueUpdateEvent(t, output);
 });
 
-test('sqsMessageRemover lambda removes message from queue when workflow succeeded', async (t) => {
+test.serial('sqsMessageRemover lambda removes message from queue and S3 when workflow succeeded', async (t) => {
+  process.env.system_bucket = randomString();
+  const message = { testdata: randomString() };
+  await createBucket(process.env.system_bucket);
+
   const sqsQueues = await createSqsQueues(randomString());
-  await awsServices.sqs().sendMessage({
-    QueueUrl: sqsQueues.queueUrl, MessageBody: JSON.stringify({ testdata: randomString() }),
+  const sqsMessage = await awsServices.sqs().sendMessage({
+    QueueUrl: sqsQueues.queueUrl, MessageBody: JSON.stringify(message),
+  }).promise();
+
+  await s3PutObject({
+    Bucket: process.env.system_bucket,
+    Key: sqsMessage.MessageId,
+    Body: JSON.stringify(sqsMessage.Body),
+  });
+
+  // Check that item exists in S3
+  const item = await s3().getObject({
```

Comment:
I elected to call `getObject` directly instead of calling `getJsonS3Object` from `@cumulus/aws-client/S3` because `getS3Object` is deprecated.